content/articles/dnsimple-plans.markdown

```diff
@@ -17,7 +17,7 @@ You'll find all [the features listed for each plan on our pricing page](https://
 
 ## Solo
 
-No base subscription fee.
+No base subscription fee. You will be charged on a pay-as-you-go basis, with fees determined by the number of active zones and their DNS query volume.
```

Comment:
The only thing I would add here (once it's finished) is a link to the query volume & fees page

Reply:
Updated and re-requested the review.

Thank you very much @itsalyse !!